ragna/deploy/_config.py

```diff
@@ -62,7 +63,8 @@ class ApiConfig(ConfigBase):
     url: str = "http://127.0.0.1:31476"
     # FIXME: this needs to be dynamic for the UI url
     origins: list[str] = ["http://127.0.0.1:31477"]
-    database_url: str = "memory"
+    # FIXME: this needs to be dynamic from the local cache root
```

Comment:
What does this FIXME mean?

Reply:
When you create a config in python and set only the `local_cache_root`, e.g.

```
from ragna.deploy import Config

config = Config(local_cache_root="/my/custom/path")
```

the default database URL will still be in `~/.cache/ragna`. Optimally, the default database URL should be `sqlite:////{local_cache_root}/ragna.db` rather than hardcoding the directory. I'll open an issue about that.

Reply:
See #326.